gh-106320: Document replacement for removed C API (#128787)
python/cpython: Doc/whatsnew/3.13.rst

    Calling this function is redundant now that :c:func:`PyFrame_GetLocals`
    returns a write-through proxy for :term:`optimized scopes <optimized scope>`.

> * Python 3.13 removed many private functions. Some of them can be replaced like
>   that:

Reviewer: ```suggestion
* Python 3.13 removed many private functions. Some of them can be replaced using these
  alternatives:
```
This sounds more natural to me.
